lib/Notification/Notifier.php

```diff
@@ -177,8 +238,14 @@ public function prepare(INotification $notification, string $languageCode): INot
 					(string) $l->t('The board "%s" has been shared with you by %s.', [$params[0], $dn])
 				);
 				$notification->setRichSubject(
-					(string) $l->t('{user} has shared the board %s with you.', [$params[0]]),
+					$l->t('{user} has shared {deck-board} with you.'),
```

Comment:
not sure if we should keep "… the board …"

Reply:
No strong opinions here, but I'd say the icon is already enough to indicate that it is deck related. Maybe @jancborchardt can comment about the general wording.